README.md

```diff
@@ -1,3 +1,8 @@
+Eclipse SmartHome now hosts the Orange implementation of a standard OSGi EnOcean base driver.

```

Comment:
That could be confusing (at least I think so), as the "that has been ..." could be point to the ESH as well.

"This repository hosts the reference implementation for the OSGi EnOcean Device Service specification, that has been originally implemented and contributed by Orange and is a sub-project of Eclipse SmartHome ."